zeppelin-interpreter/src/main/java/org/apache/zeppelin/display/AngularObject.java

```diff
@@ -56,23 +63,25 @@ public AngularObject() {
   /**
    * To create new AngularObject, use AngularObjectRegistry.add()
    *
-   * @param name name of object
-   * @param o reference to user provided object to sent to front-end
-   * @param noteId noteId belongs to. can be null
+   * @param name        name of object
+   * @param o           reference to user provided object to sent to front-end
```

Comment:
Here and below, could you please follow the project styleguide conventions and avoid horizontal alignment in code and comments?

Please read very carefully and make sure you understand [this rationale about why is that a good idea to avoid horizontal alignment](https://google.github.io/styleguide/javaguide.html#s4.6.3-horizontal-alignment)